src/view/components/formikReactSpectrum3/checkbox.jsx

```diff
@@ -33,7 +35,7 @@ const Checkbox = ({ name, description, width, ...otherProps }) => {
         onBlur={() => {
           setTouched(true);
         }}
-        validationState={error ? "invalid" : ""}
+        validationState={touched && error ? "invalid" : undefined}
```

Comment:
And I didn't really have to make this change, but I wanted it to be matchy matchy with `touched && error ? error : undefined`